src/lib/utilities.js

```diff
@@ -248,15 +252,21 @@ export const scientificNotation = (number, precision, returnHtml = true) => {
   const numPlaces = Math.abs(+exp);
 
   if ((Math.abs(number) < 1 && numPlaces > 3) || numPlaces > 9) {
-    const coefficientAndBase = `${parseFloat(toSigFigs(+val, precision))} x 10`;
+    const coefficient = parseFloat(toSigFigs(+val, precision));
 
-    return returnHtml
-      ? renderDef(`${coefficientAndBase}<sup>${+exp}</sup>`)
-      : `${coefficientAndBase}^${+exp}`;
+    if (returnFormat === 'html') {
+      return renderDef(`${coefficient} x 10<sup>${+exp}</sup>`);
+    }
+    if (returnFormat === 'latex') {
+      return `${coefficient} \\times 10^{${+exp}}`;
+    }
+    return `${coefficient} x 10^${+exp}`;
   }
 
   const sigFigged = addTheCommas(toSigFigs(number, precision));
-  return returnHtml ? renderDef(sigFigged) : sigFigged;
+  if (returnFormat === 'html') return renderDef(sigFigged);
+  if (returnFormat === 'latex') return sigFigged;
```

Comment:
unnecessary since it is returning the saem thing it would if `returnFormat` was undefined